### PR TITLE
Merge branch 'dev' of github.com:WildCodeSchool/2022-03-JS-Lille-proj…

### PR DESCRIPTION
**Pull request by Elfaram**
…ect-2-g3-movies into dev

### CODE REVIEW

**Review by ghost — APPROVED** [review]

**Review by Elfaram — COMMENTED** [review]
Don't valided this PR please ! Thx so much

**Review by LuigiMenez — APPROVED** [review]
Ok for me.

**Comment by LoicBrassart** [comment]
> Don't valided this PR please ! Thx so much

... Mistranslation, or we should *really* not validate it ? :thinking:

**Comment by Elfaram** [comment]
You can valided it but i dont think than we ´ll need it. I need to talk with my group before

**Comment by LoicBrassart** [comment]
> 
Understood, I'll review it anyway

File: frontend/src/components/QuestionSug/style.jsx [hunk 1]
```diff
@@ -1,22 +1,42 @@
...
   h2,
   h3 {
     text-align: center;
     padding-top: 1rem;
   }
-
-  .smileyImg {
-    display: flex;
-    padding: 2rem;
+  Img {
```
Comment (LoicBrassart) [inline frontend/src/components/QuestionSug/style.jsx:14]: `Img` should be `img` (the html tag) *or* `.Img` (a class name (albeit a bad one))

File: frontend/src/components/QuestionSug/style.jsx [hunk 2]
```diff
@@ -1,22 +1,42 @@
...
-    height: 125px;
-    justify-content: space-around;
+    height: auto;
+  }
+  ///// CSS Button Validation //////
+  .buttonResults {
+    display: flex;
+    justify-content: center;
+  }
+  .button {
```
Comment (LoicBrassart) [inline frontend/src/components/QuestionSug/style.jsx:28]: Plz find class names which ae not html tags, it's confusing

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Minor fixes to do

**Pull request closed without merging**